extensions/mssql/package.json

```diff
@@ -1041,10 +1041,23 @@
           "displayName": "%mssql.connectionOptions.encrypt.displayName%",
           "description": "%mssql.connectionOptions.encrypt.description%",
           "groupName": "%mssql.connectionOptions.groupName.security%",
-          "valueType": "boolean",
```

Comment:
have you tested the existing connections, please make sure we are handling them properly.

Reply:
Yes, I have, and it works well for me. Would certainly appreciate another set of eyes :)